cmd/kubeadm/app/phases/upgrade/staticpods.go

```diff
@@ -276,9 +299,18 @@ func StaticPodControlPlane(waiter apiclient.Waiter, pathMgr StaticPodPathManager
 }
 
 // rollbackOldManifests rolls back the backuped manifests if something went wrong
-func rollbackOldManifests(oldManifests map[string]string, origErr error, pathMgr StaticPodPathManager) error {
+// There is one corner case when etcd uprgade, which happens before other components, has been successfully
+// completed but api/controller-manager/scheduler experienced a problem as a result ALL manifests, etcd including
+// would be rolled back. Currently downgrade for etcd is not working and this case needs to be prevented.
+// rollbackOldManifests needs to be aware whether to restore etcd manifest or not.
+// (TODO) re-evaluate etcd downgrade story.
```

Comment:
As discussed in the meeting, we'll just skip the etcd downgrade procedure when downgrading from v1.9 to v1.8

Reply:
done